Makefile

```diff
@@ -20,50 +23,46 @@ IMAGE?=${NAME}/${STAGE}
 REPO?=${ECR}/${IMAGE}
 
 GITHUB_TOKEN=.github-token
-NPMRC=.npmrc
 BUNDLE_CFG=${HOME}/.bundle/config
-YARN_LOCK=yarn.lock
 
 all: image
 
 ${BUNDLE_CFG}: ${GITHUB_TOKEN}
-	@./bin/bundle config set rubygems.pkg.github.com ${GRP_OWNER}:`cat ${GITHUB_TOKEN}`
+	@./bin/bundle config set --local rubygems.pkg.github.com ${GRP_OWNER}:`cat ${GITHUB_TOKEN}`
```

Comment:
@ijdickinson is the `--local` flag here ok or residual from your local testing?

Reply:
It's correct. It will write to `./bundle` in the WORKDIR, rather than to the user's home directory